docs/AboutUs.md

```diff
@@ -1,52 +1,29 @@
 # About Us
+---
 
-We are a team based in the [School of Computing, National University of Singapore](http://www.comp.nus.edu.sg).
+We are a team based in the [School of Computing, National University of Singapore](http://www.comp.nus.edu.sg), doing this project for our [CS2103](http://www.comp.nus.edu.sg/~cs2103/AY1617S1/) module. We hope this app proves to be useful for people and that this app's scope is not limited solely to our module project.
 
-## Project Team
+## Team
```

Comment:
Good that you have updated your project team and place down your roles. however, some roles are a little vague. This is okay for now.